src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const source = sourceString.split(';');
+  const result = {};
+  let key = '';
+  let value = '';
+  let startKeyIndex = 0;
+  let endKeyIndex = 0;
+
+  for (let prop = 0; prop < source.length; prop++) {
+    startKeyIndex = source[prop].lastIndexOf('\n ') + 1;
```

Comment:
```suggestion
    const [key, value] = source[prop].split(':');
```

Reply:
rly, in this way it looks much simple, thanks. 
Can U tell me what does \n mean? Why did it disappear after split method?